package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Hope you don't mind this file - you can try out the `esy` sandbox build workflow on `wall` if you like.

```
npm install -g esy@next
esy install
esy build

# Then within the sandbox you can run any binary your opam package
# installs, by using "esy execute".
esy x AnyBinaryYourAppInstalls.exe
```